community/sway/etc/sway/definitions

```diff
@@ -47,6 +47,12 @@ set $rofi_theme "* {lightbg: $background-color; background: $background-color; l
 set $clipboard cliphist list | rofi -dmenu -font "$gui-font" -p "Select item to copy" -lines 10 -width 35 -theme-str $rofi_theme | cliphist decode | wl-copy
 set $clipboard-del cliphist list | rofi -dmenu -font "$gui-font" -p "Select item to delete" -lines 10 -width 35 -theme-str $rofi_theme | cliphist delete
 
+# zeit tracking
+set $zeit_list zeit list --only-projects-and-tasks --append-project-id-to-task | \
+    rofi -dmenu -font "$gui-font" -p "Select item to delete" -lines 10 -width 35 -theme-str $rofi_theme | \
+    /usr/share/sway/scripts/zeit.sh track && \
+    pkill -RTMIN+10 waybar
```

Comment:
sadly swaymsg is not waiting for a process to be finished inside (just returns immediately), so we have to put the signal inside.